modules/dreamview/frontend/src/store/hmi.js

```diff
@@ -9,6 +9,16 @@ export default class HMI {
 
     vehicles = [];
     @observable currentVehicle = 'none';
+    vehicleParam = {
```

Comment:
can we read in this value from "vehicle_config.proto"?

Reply:
Yes, the purpose of the change is to read values from vehicle_config.proto. The values defined here are default, which will be updated during initialization.